alfalfa_worker/jobs/openstudio/create_run.py

```diff
@@ -33,18 +31,18 @@ def exec(self):
         # If there are requirements.txt files in the model create a python virtual environment and install packaged there
         requirements = self.run.glob("**/requirements.txt")
         if len(requirements) > 0:
-            check_output(["python3.8", "-m", "venv", "--system-site-packages", "--symlinks", str(self.dir / '.venv')])
+            self.log_subprocess(["python3.12", "-m", "venv", "--system-site-packages", "--symlinks", str(self.dir / '.venv')])
```

Comment:
`log_subprocess` takes the output of the subprocess and logs it. This makes logs more useful as they include more of the output from the job.

Reply:
That's a good point. I think this was an artifact from when I had multiple python versions installed at once.

Reply:
Pls either make the update for this PR or add an issue.